uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -63,6 +63,22 @@ function hrcore_civicrm_container($container) {
   $loader->load('config/container/container.xml');
 }
 
+/**
+ * Implements hook_civicrm_buildForm
+ *
+ * @param string $formName
+ * @param object $form
+ */
+function hrcore_civicrm_buildForm($formName, &$form) {
+  if($formName == 'CRM_Activity_Form_ActivityFilter') {
+    _hrcore_modifyExcludeAndIncludeActivityTypeFields($form);
+  }
+
+  if($formName == 'CRM_Activity_Form_ActivityLinks') {
+    _hrcore_filterActivityTypeLinks($form);
+  }
```

Comment:
Let's do here the same thing that was done for `hrcore_civicrm_pre` [here](https://github.com/civicrm/civihr/blob/cae79b10a436aecc88efe92b42d5ecb84993ffbb/uk.co.compucorp.civicrm.hrcore/hrcore.php#L200-L208) and extract the logic in `_hrcore_modifyExcludeAndIncludeActivityTypeFields` and `_hrcore_filterActivityTypeLinks` to classes.